### ISSUE TITLE
app-tap/tarantoolctl.test.lua fails occasionally on OS X

### ISSUE THREAD

**Issue by locker** [tarantool/tarantool#3557]
Reproduced by [Travis CI](https://travis-ci.org/tarantool/tarantool/jobs/406401510):
```
app-tap/tarantoolctl.test.lua                                   [ fail ]
Test failed! Last 15 lines of the result file:
    1..6
    ok - check 'start' command status for 'filler'
    ok - check lsn before
    ok - execution result
    ok - check lsn after
    ok - execution result
    ok - check lsn after
    # fill and test play output: end
ok - fill and test play output
# failed subtest: 1
Last 15 lines of Tarantool Log file [Instance "app_server"][/Users/travis/build/tarantool/tarantool/test/var/001_app-tap/tarantoolctl.test.lua.tarantool.log]:
Netbox console API is deprecated since 1.10, please use require('console').connect() instead
Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
Netbox console API is deprecated since 1.10, please use require('console').connect() instead
Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
Netbox console API is deprecated since 1.10, please use require('console').connect() instead
Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
make: *** [test_osx] Error 255
```

**Comment by Totktonada** [comment]
@locker Pushed new version of test-run (printing whole reject file, tarantool/test-run#102) into tarantool (1.9, 1.10, 2.0).

**Comment by stek29** [comment]
@Totktonada here's output of failing test with new version of `test-run`:

<details><summary>log</summary>

```
[006] Test failed! Output from reject file app-tap/tarantoolctl.reject:
[006] TAP version 13
[006] 1..6
[006]     # basic test
[006]     1..16
[006]     ok - check 'start' command status for 'script'
[006]     ok - check 'start' stderr for 'script'
[006]     ok - check 'status' command status for 'script'
[006]     ok - check 'status' stderr for 'script'
[006]     ok - check 'start' command status for 'script'
[006]     ok - check 'start' stderr for 'script'
[006]     ok - check 'status' command status for 'script'
[006]     ok - check 'status' stderr for 'script'
[006]     ok - check 'stop' command status for 'script'
[006]     ok - check 'stop' stderr for 'script'
[006]     ok - check 'status' command status for 'script'
[006]     ok - check 'status' stderr for 'script'
[006]     ok - check 'stop' command status for 'script'
[006]     ok - check 'stop' stderr for 'script'
[006]     ok - check 'status' command status for 'script'
[006]     ok - check 'status' stderr for 'script'
[006]     # basic test: end
[006] ok - basic test
[006]     # basic test for bad script
[006]     1..8
[006]     ok - check 'start' command status for 'script'
[006]     ok - check 'start' stderr for 'script'
[006]     ok - check 'start' command status for 'bad_script'
[006]     ok - check 'start' stderr for 'bad_script'
[006]     ok - check 'start' command status for 'good_script'
[006]     ok - check 'eval' command status for 'good_script bad_script.lua'
[006]     ok - check 'eval' stderr for 'good_script bad_script.lua'
[006]     ok - check 'stop' command status for 'good_script'
[006]     # basic test for bad script: end
[006] ok - basic test for bad script
[006]     # check answers in case of call
[006]     1..6
[006]     ok - check 'start' command status for 'good_script'
[006]     ok - check 'eval' command status for 'good_script bad_script.lua'
[006]     ok - check 'eval' stderr for 'good_script bad_script.lua'
[006]     ok - check 'eval' command status for 'good_script ok_script.lua'
[006]     ok - check 'eval' stdout for 'good_script ok_script.lua'
[006]     ok - check 'stop' command status for 'good_script'
[006]     # check answers in case of call: end
[006] ok - check answers in case of call
[006]     # check basic help
[006]     1..4
[006]     ok - check stderr of 'tarantoolctl' without config
[006]     ok - check stderr of 'tarantoolctl help' without config
[006]     ok - check stderr of 'tarantoolctl --help' without config
[006]     ok - check stderr of 'tarantoolctl' with config
[006]     # check basic help: end
[006] ok - check basic help
[006]     # fill and test cat output
[006]     1..29
[006]     ok - check 'start' command status for 'filler'
[006]     ok - cat result
[006]     ok - cat line count
[006]     ok - cat + --space=512 result
[006]     ok - cat + --space=512 line count
[006]     ok - cat + --space=666 result
[006]     ok - cat + --space=666 line count
[006]     ok - cat + --show-system result
[006]     ok - cat + --show-system line count
[006]     ok - cat + --format=json result
[006]     ok - cat + --format=json line count
[006]     ok - cat + --format=lua result
[006]     ok - cat + --format=lua line count
[006]     ok - cat + --from=3 --to=6 --format=json result
[006]     ok - cat + --from=3 --to=6 --format=json line count
[006]     ok - cat + --from=3 --to=6 --format=json --show-system result
[006]     ok - cat + --from=3 --to=6 --format=json --show-system line count
[006]     ok - cat + --from=6 --to=3 --format=json --show-system result
[006]     ok - cat + --from=6 --to=3 --format=json --show-system line count
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 result
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 line count
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 --replica 2 result
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 --replica 2 line count
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 2 result
[006]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 2 line count
[006]     ok - cat + --space=280 result
[006]     not ok - cat + --space=280 line count
[006]       ---
[006]       filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]       trace:
[006]       - line: 316
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat: upvalue
[006]         name: check_ctlcat_snap
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: 326
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat: local
[006]         name: fun
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: 218
[006]         source: '@builtin/tap.lua'
[006]         filename: builtin/tap.lua
[006]         what: Lua
[006]         namewhat: method
[006]         name: test
[006]         src: builtin/tap.lua
[006]       - line: 325
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat:
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: -1
[006]         source: =[C]
[006]         filename: eval
[006]         what: C
[006]         namewhat: global
[006]         name: pcall
[006]         src: '[C]'
[006]       - line: 0
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: main
[006]         namewhat:
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       line: 0
[006]       expected: 21
[006]       got: 22
[006]       ...
[006]     ok - cat + --space=288 result
[006]     not ok - cat + --space=288 line count
[006]       ---
[006]       filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]       trace:
[006]       - line: 316
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat: upvalue
[006]         name: check_ctlcat_snap
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: 326
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat: local
[006]         name: fun
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: 218
[006]         source: '@builtin/tap.lua'
[006]         filename: builtin/tap.lua
[006]         what: Lua
[006]         namewhat: method
[006]         name: test
[006]         src: builtin/tap.lua
[006]       - line: 325
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: Lua
[006]         namewhat:
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       - line: -1
[006]         source: =[C]
[006]         filename: eval
[006]         what: C
[006]         namewhat: global
[006]         name: pcall
[006]         src: '[C]'
[006]       - line: 0
[006]         source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]         filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]         what: main
[006]         namewhat:
[006]         src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]       line: 0
[006]       expected: 47
[006]       got: 49
[006]       ...
[006]     # fill and test cat output: end
[006] not ok - failed subtests
[006]   ---
[006]   filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]   trace:
[006]   - line: 244
[006]     source: '@builtin/tap.lua'
[006]     filename: builtin/tap.lua
[006]     what: Lua
[006]     namewhat: method
[006]     name: test
[006]     src: builtin/tap.lua
[006]   - line: 325
[006]     source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]     filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]     what: Lua
[006]     namewhat:
[006]     src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]   - line: -1
[006]     source: =[C]
[006]     filename: eval
[006]     what: C
[006]     namewhat: global
[006]     name: pcall
[006]     src: '[C]'
[006]   - line: 0
[006]     source: '@/Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]     filename: /Users/stek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua
[006]     what: main
[006]     namewhat:
[006]     src: '...ek/Projects/tarantool/test/app-tap/tarantoolctl.test.lua'
[006]   planned: 29
[006]   failed: 2
[006]   line: 0
[006]   ...
[006]     # fill and test play output
[006]     1..6
[006]     ok - check 'start' command status for 'filler'
[006]     ok - check lsn before
[006]     ok - execution result
[006]     ok - check lsn after
[006]     ok - execution result
[006]     ok - check lsn after
[006]     # fill and test play output: end
[006] ok - fill and test play output
[006] # failed subtest: 1
[006]
[006] Last 15 lines of Tarantool Log file [Instance "app_server"][/Users/stek/Projects/tarantool/build/test/var/006_app-tap/tarantoolctl.test.lua.tarantool.log]:
[006] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[006] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
[006] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[006] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
[006] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[006] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
```

</details>

**Comment by locker** [comment]
Another failure got caught by [Travis CI](https://travis-ci.org/tarantool/tarantool/jobs/416077441).
<details><summary>Detailed log</summary>

```
[004] app-tap/tarantoolctl.test.lua                                   [ fail ]
[004] Test failed! Output from reject file app-tap/tarantoolctl.reject:
[004] TAP version 13
[004] 1..6
[004]     # basic test
[004]     1..16
[004]     ok - check 'start' command status for 'script'
[004]     ok - check 'start' stderr for 'script'
[004]     ok - check 'status' command status for 'script'
[004]     ok - check 'status' stderr for 'script'
[004]     ok - check 'start' command status for 'script'
[004]     ok - check 'start' stderr for 'script'
[004]     ok - check 'status' command status for 'script'
[004]     ok - check 'status' stderr for 'script'
[004]     ok - check 'stop' command status for 'script'
[004]     ok - check 'stop' stderr for 'script'
[004]     not ok - check 'status' command status for 'script'
[004]       ---
[004]       filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]       trace:
[004]       - line: 125
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: upvalue
[004]         name: check_ok
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: 163
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: local
[004]         name: fun
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: 218
[004]         source: '@builtin/tap.lua'
[004]         filename: builtin/tap.lua
[004]         what: Lua
[004]         namewhat: method
[004]         name: test
[004]         src: builtin/tap.lua
[004]       - line: 162
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: 
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: -1
[004]         source: =[C]
[004]         filename: eval
[004]         what: C
[004]         namewhat: global
[004]         name: pcall
[004]         src: '[C]'
[004]       - line: 0
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: main
[004]         namewhat: 
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       line: 0
[004]       expected: 1
[004]       got: 2
[004]       ...
[004]     not ok - check 'status' stderr for 'script'
[004]       ---
[004]       filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]       line: 0
[004]       trace:
[004]       - line: 125
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: upvalue
[004]         name: check_ok
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: 163
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: local
[004]         name: fun
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: 218
[004]         source: '@builtin/tap.lua'
[004]         filename: builtin/tap.lua
[004]         what: Lua
[004]         namewhat: method
[004]         name: test
[004]         src: builtin/tap.lua
[004]       - line: 162
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: Lua
[004]         namewhat: 
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       - line: -1
[004]         source: =[C]
[004]         filename: eval
[004]         what: C
[004]         namewhat: global
[004]         name: pcall
[004]         src: '[C]'
[004]       - line: 0
[004]         source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]         filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]         what: main
[004]         namewhat: 
[004]         src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]       ...
[004] Expected to find 'is stopped' in 'Pid file exists, but the control socket (./script.control) doesn't
[004] '
[004] 2		Pid file exists, but the control socket (./script.control) doesn't
[004] 
[004]     ok - check 'stop' command status for 'script'
[004]     ok - check 'stop' stderr for 'script'
[004]     ok - check 'status' command status for 'script'
[004]     ok - check 'status' stderr for 'script'
[004]     # basic test: end
[004] not ok - failed subtests
[004]   ---
[004]   filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]   trace:
[004]   - line: 244
[004]     source: '@builtin/tap.lua'
[004]     filename: builtin/tap.lua
[004]     what: Lua
[004]     namewhat: method
[004]     name: test
[004]     src: builtin/tap.lua
[004]   - line: 162
[004]     source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]     filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]     what: Lua
[004]     namewhat: 
[004]     src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]   - line: -1
[004]     source: =[C]
[004]     filename: eval
[004]     what: C
[004]     namewhat: global
[004]     name: pcall
[004]     src: '[C]'
[004]   - line: 0
[004]     source: '@/Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]     filename: /Users/travis/build/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua
[004]     what: main
[004]     namewhat: 
[004]     src: '...d/tarantool/tarantool/test/app-tap/tarantoolctl.test.lua'
[004]   planned: 16
[004]   failed: 2
[004]   line: 0
[004]   ...
[004]     # basic test for bad script
[004]     1..8
[004]     ok - check 'start' command status for 'script'
[004]     ok - check 'start' stderr for 'script'
[004]     ok - check 'start' command status for 'bad_script'
[004]     ok - check 'start' stderr for 'bad_script'
[004]     ok - check 'start' command status for 'good_script'
[004]     ok - check 'eval' command status for 'good_script bad_script.lua'
[004]     ok - check 'eval' stderr for 'good_script bad_script.lua'
[004]     ok - check 'stop' command status for 'good_script'
[004]     # basic test for bad script: end
[004] ok - basic test for bad script
[004]     # check answers in case of call
[004]     1..6
[004]     ok - check 'start' command status for 'good_script'
[004]     ok - check 'eval' command status for 'good_script bad_script.lua'
[004]     ok - check 'eval' stderr for 'good_script bad_script.lua'
[004]     ok - check 'eval' command status for 'good_script ok_script.lua'
[004]     ok - check 'eval' stdout for 'good_script ok_script.lua'
[004]     ok - check 'stop' command status for 'good_script'
[004]     # check answers in case of call: end
[004] ok - check answers in case of call
[004]     # check basic help
[004]     1..4
[004]     ok - check stderr of 'tarantoolctl' without config
[004]     ok - check stderr of 'tarantoolctl help' without config
[004]     ok - check stderr of 'tarantoolctl --help' without config
[004]     ok - check stderr of 'tarantoolctl' with config
[004]     # check basic help: end
[004] ok - check basic help
[004]     # fill and test cat output
[004]     1..29
[004]     ok - check 'start' command status for 'filler'
[004]     ok - cat result
[004]     ok - cat line count
[004]     ok - cat + --space=512 result
[004]     ok - cat + --space=512 line count
[004]     ok - cat + --space=666 result
[004]     ok - cat + --space=666 line count
[004]     ok - cat + --show-system result
[004]     ok - cat + --show-system line count
[004]     ok - cat + --format=json result
[004]     ok - cat + --format=json line count
[004]     ok - cat + --format=lua result
[004]     ok - cat + --format=lua line count
[004]     ok - cat + --from=3 --to=6 --format=json result
[004]     ok - cat + --from=3 --to=6 --format=json line count
[004]     ok - cat + --from=3 --to=6 --format=json --show-system result
[004]     ok - cat + --from=3 --to=6 --format=json --show-system line count
[004]     ok - cat + --from=6 --to=3 --format=json --show-system result
[004]     ok - cat + --from=6 --to=3 --format=json --show-system line count
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 result
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 line count
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 --replica 2 result
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 1 --replica 2 line count
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 2 result
[004]     ok - cat + --from=3 --to=6 --format=json --show-system --replica 2 line count
[004]     ok - cat + --space=280 result
[004]     ok - cat + --space=280 line count
[004]     ok - cat + --space=288 result
[004]     ok - cat + --space=288 line count
[004]     # fill and test cat output: end
[004] ok - fill and test cat output
[004]     # fill and test play output
[004]     1..6
[004]     ok - check 'start' command status for 'filler'
[004]     ok - check lsn before
[004]     ok - execution result
[004]     ok - check lsn after
[004]     ok - execution result
[004]     ok - check lsn after
[004]     # fill and test play output: end
[004] ok - fill and test play output
[004] # failed subtest: 1
[004] 
[004] Last 15 lines of Tarantool Log file [Instance "app_server"][/Users/travis/build/tarantool/tarantool/test/var/004_app-tap/tarantoolctl.test.lua.tarantool.log]:
[004] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[004] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
[004] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[004] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
[004] Netbox console API is deprecated since 1.10, please use require('console').connect() instead
[004] Netbox text protocol support is deprecated since 1.10, please use require('console').connect() instead
[Main process] Got failed test; gently terminate all workers...
[004] Worker "004_app-tap" got failed test; stopping the server...
---------------------------------------------------------------------------------
Statistics:
* fail: 1
* skip: 1
* pass: 102
Failed tasks:
- [app-tap/tarantoolctl.test.lua, null]
# logfile:        /Users/travis/build/tarantool/tarantool/test/var/log/004_app-tap.log
# reproduce file: /Users/travis/build/tarantool/tarantool/test/var/reproduce/004_app-tap.list.yaml
---
- [app-tap/clock.test.lua, null]
- [app-tap/table.test.lua, null]
- [app-tap/string.test.lua, null]
- [app-tap/msgpackffi.test.lua, null]
- [app-tap/inspector.test.lua, null]
- [app-tap/iconv.test.lua, null]
- [app-tap/errno.test.lua, null]
- [app-tap/msgpack.test.lua, null]
- [app-tap/logger_pipe.test.lua, null]
- [app-tap/console.test.lua, null]
- [app-tap/csv.test.lua, null]
- [app-tap/init_script.test.lua, null]
- [app-tap/info.test.lua, null]
- [app-tap/json.test.lua, null]
- [app-tap/tap.test.lua, null]
- [app-tap/pwd.test.lua, null]
- [app-tap/logger.test.lua, null]
- [app-tap/uri.test.lua, null]
- [app-tap/minimal.test.lua, null]
- [app-tap/trigger.test.lua, null]
- [app-tap/http_client.test.lua, null]
- [app-tap/tarantoolctl.test.lua, null]
...
```
</details>

**Comment by stek29** [comment]
Tests are now passing for me, but I had to run `make test` three times.

First time it failed on workers becoming unresponsive, then it failed on backup test with `TarantoolStartError`, and then in onreplace2 with "Instance bootstrap hasn't finished yet" & "User cluster not found".

**Comment by locker** [comment]
Fixed by 9bf93573fdf8b669f7ebe70b2934ef12ac025aa7
